UI/src/model/PointAssetsCollection.js

```diff
@@ -1,17 +1,61 @@
 (function(root) {
   root.PointAssetsCollection = function(backend, endPointName) {
     var isComplementaryActive = false;
+    var isTrafficSigns = (endPointName == 'trafficSigns');
+    var trafficSignsShowing = {
+      speedLimits: false, //[1, 2, 3, 4, 5, 6]
+      pedestrianCrossings: false, //[7]
+      maximumLengths: false, //[8]
+      generalWarnings: false, //[9]
+      turningRestrictions: false //[10, 11, 12]
+    };
+
+    var trafficSignValues = {
+      speedLimits: [1, 2, 3, 4, 5, 6],
+      pedestrianCrossings: [7],
+      maximumLengths: [8],
+      generalWarnings: [9],
+      turningRestrictions: [10, 11, 12]
+    };
+
 
     var filterComplementaries = function (assets) {
       if(isComplementaryActive)
         return assets;
       return _.where(assets, {linkSource: 1});
     };
 
+     var filterTrafficSigns = function (asset) {
+       return _.filter(asset, function (asset) {
+         var existingValue = _.first(_.find(asset.propertyData, function(prop){return prop.publicId === "trafficSigns_type";}).values);
```

Comment:
The `_.first` usage is not necessary since the `_.find` already finds the first element that verifies the condition `return prop.publicId === "trafficSigns_type";`